solvers/callcdcs.m

```diff
@@ -25,36 +25,51 @@
     [F_struc,K] = fix1Dqcone(F_struc,K);
 end
 
-options.admmpdcp.verbose = options.verbose;
+options.cdcs.verbose = options.verbose;
 aK.f = K.f;
 aK.l = K.l;
 aK.q = K.q;
 aK.s = K.s;
 K = aK;
 
 if options.savedebug
-    save admmpdcp F_struc c K pars
+    save cdcs F_struc c K pars
```

Comment:
This should be cdcsdebug (the template code I wrote was wrong, all debug files are called *solver*debug)

There is no variable called pars in the work space. Should be the options structure sent, so you'll have to create one first from options.cdcs